libs/application/templates/parental-leave/src/fields/Rights/GiveDaysSlider.tsx

```diff
@@ -0,0 +1,101 @@
+import React, { FC, useState } from 'react'
+import { FieldBaseProps, getValueViaPath } from '@island.is/application/core'
+import { Controller, useFormContext } from 'react-hook-form'
+import { Box, Text } from '@island.is/island-ui/core'
+import { theme } from '@island.is/island-ui/theme'
+import { useLocale } from '@island.is/localization'
+import { m } from '../../lib/messages'
+import Slider from '../components/Slider'
+import BoxChart, { BoxChartKey } from '../components/BoxChart'
+
+const GiveDaysSlider: FC<FieldBaseProps> = ({ field, application }) => {
+  const maxDays = 45
```

Comment:
Should `maxDays` and `maxDaysToGive` be the same variable?

Reply:
Good catch thanks should be fixed now.